Sources_v3/StreamChatUI/MessageActionsPopup/MessageActionsView+ActionButton.swift

```diff
@@ -26,14 +27,28 @@ extension MessageActionsView {
         }
         
         override open func updateContent() {
-            setImage(actionItem?.icon, for: .normal)
-            tintColor = actionItem?.isDestructive == true ? .systemRed : .darkGray
-            imageView?.tintColor = tintColor
+            guard let name = actionItem?.name else {
+                setImage(nil, for: .normal)
+                setTitle(nil, for: .normal)
+                return
+            }
 
-            let titleColor: UIColor = actionItem?.isDestructive == true ? .systemRed : .black
-            setTitle(actionItem?.title, for: .normal)
-            titleLabel?.font = UIFont.preferredFont(forTextStyle: .headline).bold
-            setTitleColor(titleColor, for: .normal)
+            setImage(
+                uiConfig.messageList.messageActionsSubviews.actionIcons[name],
+                for: .normal
+            )
```

Comment:
Shouldn't we log some error if we don't find the image?

Reply:
not relevant since the icon is now part of `ChatMessageAction`